CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ ENHANCEMENTS:
 
 * cmd/tfplugindocs: Use existing Terraform CLI binary if available on PATH, otherwise download latest Terraform CLI binary (https://github.com/hashicorp/terraform-plugin-docs/pull/124)
 * cmd/tfplugindocs: Added `tf-version` flag for specifying Terraform CLI binary version to download, superseding the PATH lookup (https://github.com/hashicorp/terraform-plugin-docs/pull/124)
+* template functions: Added the `split` command to split a string into substrings
```

Comment:
Sorry, I think I should have been more explicit.

The changelog format is defined here https://www.terraform.io/plugin/sdkv2/best-practices/versioning#changelog-specification .

Your change will need a new release, probably will be a `0.8.0`, as it's a net new feature, but it's additive so it doesn't warrant a major release.

Reply:
Apologies @detro, see the next commit